### PR TITLE
Rename variable according to #555

### PR DESCRIPTION
**Pull request by pBouillon**
Rename variables in the `IRandom` interface according to #555

### CODE REVIEW

**Comment by pBouillon** [comment]
@dahlia I'm having troubles understanding what is expected to pass the _changelog_ check, could you help me ?

**Comment by pBouillon** [comment]
> 
> 
> > @dahlia I'm having troubles understanding what is expected to pass the changelog check, could you help me ?
> 
> @pBouillon You need to write a log about this change into "Backward-incompatible interface changes" section in the _CHANGES.md_ file. It should be like:
> 
> ```gfm
>  -  Renamed `minValue`/`maxValue` parameters to `lowerBound`/`upperBound`
>     of `IRandom.Next()` methods.  [[#555], [#558]]
> ```
> 
> Also there should be reference definitions for `[#555]` and `[#558]`, but you could get how to do that if you look around at other changelogs.

Thank you ! It's done with the relevant links to the issue and this PR

**Comment by pBouillon** [comment]
@dahlia All tests clear ! I also rebased my fork from the remote master branch, do I need to add anything else to this PR ? 😄

**Comment by dahlia** [comment]
@pBouillon Sorry, but could you rebase your changes on the current master and move your changelogs from 0.6.0 to 0.7.0?

**Comment by pBouillon** [comment]
Of course @dahlia, rebase done !

**Comment by dahlia** [comment]
@pBouillon We had just release new version (0.6.0) few hours ago, so your change should be included in the next release (0.7.0).  So could you please move changelogs you wrote from <q>Version 0.6.0</q> to <q>Version 0.7.0</q>?

**Comment by pBouillon** [comment]
All set @dahlia, let me know if you would like any other modification

File: CHANGES.md [hunk 1]
```diff
@@ -4,10 +4,12 @@ Libplanet changelog
 Version 0.7.0
 -------------
 
-To be released.
+Released on October 1, 2019.
```
Comment (dahlia) [inline CHANGES.md]: Oh, could you left date this line as it was (i.e., <q>To be released.</q>)?  We should change this date when we actually make a new release (about a month later).

Comment (pBouillon) [reply CHANGES.md]: Sure thing !

Comment (pBouillon) [reply CHANGES.md]: Done !

**Comment by dahlia** [comment]
@pBouillon Okay, everything seems alright! We are going to merge this patch immediately after the build passes.  Thanks you for the contribution and following our picky requests.  🙏

**Comment by pBouillon** [comment]
Alright ! Thank you for your reactivity about this PR ! 😃

**Pull request merged**